### PR TITLE
Update plugin for TF2 models support

### PR DESCRIPTION
**Pull request by nicolasdalsass**
Replace keras2onnx with tf2onnx, and handle cross-version compatible calls
to have tf1 and tf2 models support.
Update README to reference up-to-date DSS presets instead of a static package listing.

### CODE REVIEW

File: code-env/python/desc.json [hunk 1]
```diff
@@ -1,5 +1,5 @@
 {
-  "acceptedPythonInterpreters": ["PYTHON36", "PYTHON35", "PYTHON34"],
+  "acceptedPythonInterpreters": ["PYTHON37"],
```
Comment (arnaudde) [inline code-env/python/desc.json:2]: Why don't we allow py38 and py39 ?

Comment (arnaudde) [inline code-env/python/desc.json:2]: Why don't we allow py38 and py39 ?

Comment (nicolasdalsass) [reply code-env/python/desc.json:2]: I hadn't thought about them, but I guess why not ? I'm going to try them out !

**Review by arnaudde — COMMENTED** [review]
Code looks good 👍 
Tested with py37 through the `TestDataGeneratorOnnxExporter` with 11.0.0 visual dl packages (ie tf2) and with the following set of package with tf1
```
tensorflow>=1.15,<2.0
keras==2.3.1
keras-preprocessing==1.1.0
scikit-learn>=0.20,<0.21
scipy>=1.2,<1.3
statsmodels>=0.10,<0.11
jinja2>=2.10,<2.11
MarkupSafe<2.1.0
flask>=1.0,<1.1
h5py==2.10.0
pillow==6.2.2
cloudpickle>=1.3,<1.6
```

**Comment by nicolasdalsass** [comment]
Small modifs to have py38 and py39 compat. 
py310 fails horribly for some reason I didn't dig into, I don't think it's needed yet anyway ^^

**Review by arnaudde — APPROVED** [review]
Tested again with py 3.9.12 and 3.7.13 works well

**Pull request merged**